cmd/podman/search.go

```diff
@@ -101,10 +106,11 @@ func searchCmd(c *cli.Context) error {
 
 	format := genSearchFormat(c.String("format"))
 	opts := searchOpts{
-		format:  format,
-		noTrunc: c.Bool("no-trunc"),
-		limit:   c.Int("limit"),
-		filter:  c.StringSlice("filter"),
+		format:     format,
+		noTrunc:    c.Bool("no-trunc"),
+		limit:      c.Int("limit"),
+		filter:     c.StringSlice("filter"),
+		skipVerify: !c.BoolT("tls-verify"),
```

Comment:
I think in most of our code we call a flag like this tlsVerify and then do the inverse of it's value when we set DockerInsecureSkipTLSVerify.